index.js

```diff
@@ -372,15 +372,13 @@ class K8sExecutor extends Executor {
             const updateConfig = {
                 apiUri: this.ecosystem.api,
                 buildId,
-                token
+                token,
+                stats: {
```

Comment:
will this override entire `stats` field (including `hostname`)?

Reply:
`stats` field will be merged.
Thus existed key (`imagePullStartTime`) will not drop when hostname update.

https://github.com/screwdriver-cd/screwdriver/blob/8c199b0b44ba9c9a663be3e4785f4b08aee9e4db/plugins/builds/update.js#L190C21-L193